deploy/README.md

```diff
@@ -162,6 +167,7 @@ Following is the list of resource-groups and resources that should be created if
 
     - Storage account named `rawdata<6-character-random-string>` to store raw input data for pipelines.
     - Keyvault named `aoi-data-kv` to store credentials as secrets.
+    - Postgres Single Server DB named `aoi-pg-server`
 
 - `aoi-monitor-rg`
 
```

Comment:
Can we add a section on where to find the results of the detection model? Table name and may be a single query to see / check the detection in Postgres from the Portal.